docs/source/tutorials/configure_auto_optimizer.rst

```diff
@@ -0,0 +1,262 @@
+.. _how_to_configure_auto_optimizer:
```

Comment:
Seems this auto optimizer only supports onnxruntime optimization with onnx model. Can we add this note to the documentation? For example, this auto optimizer doesn't support openvino model and snpe model optimization.